integration-tests/features/environment.py

```diff
@@ -1,4 +1,11 @@
 import contextlib
+import subprocess
+
+def before_all(context):
+    # Some steps require sudo, so for convenience in interactive use,
+    # we ensure we prompt for elevated permissions immediately,
+    # rather than potentially halting midway through a test
+    subprocess.check_output(["sudo", "echo", "Elevated permissions needed"])
```

Comment:
Alternatively, if we go the DBus option, we may be able to get away from requiring sudo on the client in the first place: https://github.com/leapp-to/prototype/issues/7